src/main/java/kitchenpos/application/tablegroup/TableGroupEventHandler.java

```diff
@@ -0,0 +1,49 @@
+package kitchenpos.application.tablegroup;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import kitchenpos.domain.order.OrderRepository;
+import kitchenpos.domain.order.OrderStatus;
+import kitchenpos.domain.table.OrderTable;
+import kitchenpos.domain.table.OrderTableRepository;
+import kitchenpos.domain.tablegroup.TableGroupGroupedEvent;
+import kitchenpos.domain.tablegroup.TableGroupUngroupedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TableGroupEventHandler {
+
+    private final OrderRepository orderRepository;
+    private final OrderTableRepository orderTableRepository;
+
+    public TableGroupEventHandler(final OrderRepository orderRepository,
+                                  final OrderTableRepository orderTableRepository) {
+        this.orderRepository = orderRepository;
+        this.orderTableRepository = orderTableRepository;
+    }
+
+    @EventListener(TableGroupGroupedEvent.class)
+    public void handleGroup(final TableGroupGroupedEvent event) {
```

Comment:
파라미터에 Event가 선언되어있으면 ArgumentResolver로 인식합니다!
크게 상관은 없지만 명시적으로 선언해준 이유가 있을까요?

```suggestion
    @EventListener
    public void handleGroup(final TableGroupGroupedEvent event) {
```

Reply:
오.. 이 부분은 솔직히 시간이 부족해서 제가 제대로 모르고 사용한 것 같아요 🥲

이벤트를 발행하는 방식 말고 다른 방식을 사용할까 생각하다가 한 번 써보자!라는 마인드로 썼는데 하하,,

하나 더 알아갑니당